include/openspace/util/coordinateconversion.h

```diff
@@ -26,18 +26,49 @@
 #define __OPENSPACE_CORE___COORDINATECONVERSION___H__
 
 #include <ghoul/glm.h>
+#include <string>
 
 namespace openspace {
 
 /**
- * Converts from ICRS coordinates to galactic cartesian coordinates.
+ * Converts from ICRS decimal degrees coordinates to galactic cartesian coordinates.
  * \param ra Right ascension, given in decimal degrees
  * \param dec Declination, given in decimal degrees
  * \param distance The distance, or radius, to the position given in any unit.
+ * \param isDegrees Boolean to say if the incoming ra and dec are in degrees or radians
  * \return A position in galactic cartesian coordinates, given in the same unit as the
  *         distance parameter.
  */
-glm::dvec3 icrsToGalacticCartesian(float ra, float dec, double distance);
+glm::dvec3 icrsToGalacticCartesian(double ra, double dec, double distance,
+    bool isDegrees = true);
```

Comment:
Is it really necessary to have the support for changing degrees to radians in the function header?  Might be nicer to just support radians and then have an extra function (that probably already exists in Lua) that converts from degrees to radians.

Less to think about when trying to read:
`icrsToGalacticCartesian(10.0, 2.0, 1.0, true)`   and figuring out what the last parameter is
`icrsToGalacticCartesian(glm::radians(10.0), glm::radians(2.0), 1.0);`  would be easier to read imho

Reply:
Sure, but then I would suggest to default to degrees instead since that is what I have seen is more common for Ra and Dec coordinates.